content/verification/veteran-verification/release-notes/2023-05-18

```diff
@@ -0,0 +1,26 @@
+We added a new version (v2) of the Veteran Verification API to the developer portal, and the API has been renamed to the 'Veteran Service History and Eligibility API'.
+
+The following enhancements and new features are available in v2:  
+
+All of the API's endpoints can now be accessed using either Authorization Code Flow OAuth or Client Credentials Grant (CCG) OAuth.   
+There are two versions of each endpoint, a GET when ICN is provided and a POST when demographic details will be used to match a Veteran.   
+The disability_rating endpoint now includes the diagnostic code and name associated with each individual rating.
+The service_history endpoint now includes additional information:
+A service episode's component_of_service identifies unique sub-categories within a main branch of service, such as "Reserves."  In the prior version (Veteran Verification, v1), the branch_of_service is consistently a main branch (e.g., Army, Navy...) with no sub-category values such as "Reserves."  Adding this distinct field for this purpose clarifies which branch of service a period of Reserve duty is associated with.  
+A new military summary section includes information pulled from all service episodes, such as:
+
+An indicator of whether the person is currently on active duty.
+
+An indicator of whether the person ever served in combat duty.
+
+The total number of Regular Active Duty Days, Reserve Active Duty Days, Guard Active Duty Days, and Training Days. 
+
+The new flashes endpoint shares an attribute on a Veteran's profile that describes special circumstances which can apply to a Veteran. Flashes can aid in properly routing or expediting certain claims or determining eligibility for some VA benefits. This endpoint currently shares only a limited subset of the available flashes that can be assigned to Veterans. If you have a use case that would benefit from additional flashes being added to this endpoint, please get in touch with us with the details. 
+The new enrolled_benefits endpoint shares the codes and names of benefits a Veteran is enrolled in.
+Changes to follow current standards, such as improvements to error details and HTTPS response codes used.
+As always, it's important to know that the new version must support backward compatibility.  It should:
+
+Be prepared to handle HTTP status codes not explicitly specified in endpoint definitions.
+Be tolerant of unknown fields in the payload and ignore the deserialization of new fields.
+Follow the redirect when the server returns an HTTP status code 301 (Moved Permanently).
+Review the API documentation to learn more about the new version of this API.
```

Comment:
'API documentation' will link to the new Veteran Service History and Eligibility API documentation?

Reply:
Linking to this thread: https://lighthouseva.slack.com/archives/C03TPJWJ0E6/p1685046088505399?thread_ts=1684353511.797529&cid=C03TPJWJ0E6